text-serializer-legacy/src/main/java/net/kyori/adventure/text/serializer/legacy/LegacyComponentSerializerImpl.java

```diff
@@ -471,6 +488,7 @@ private void applyFullFormat() {
     private ComponentFlattener flattener = ComponentFlattener.basic();
 
     BuilderImpl() {
+      BUILDER.accept(this); // let service provider touch the builder before anybody else touches it
```

Comment:
Specifically this change could irreversibly affect behavior.

Reply:
This only provides defaults - anything can be changed by using the builder methods like normal.